tests/vm_test.go

```diff
@@ -30,6 +30,8 @@ import (
 	"strings"
 	"time"
 
+	"kubevirt.io/kubevirt/tests/storage"
```

Comment:
I don't think we want to go the cross-sig-import route

I have a PR open (although with some resistance) to expose a vmdv factory
https://github.com/kubevirt/kubevirt/pull/11670